node/internal/actors/direct_start_agent.go

```diff
@@ -87,6 +87,28 @@ func (a *DirectStartAgent) Receive(ctx *goakt.ReceiveContext) {
 		ctx.Response(&actorproto.LameDuckResponse{
 			Success: true,
 		})
+	case *actorproto.PingWorkload:
+		a.logger.Debug("PingWorkload received", slog.String("name", ctx.Self().Name()), slog.String("workload", m.WorkloadId))
```

Comment:
we should probably refactor this into a function so the case statement arms don't get obscured.

Reply:
I knew you were going to say this....

Reply:
Just call me `kevlint`